Tests/NotifyTests/Mocks/MockNotifyStoring.swift

```diff
@@ -24,6 +29,10 @@ class MockNotifyStoring: NotifyStoring {
         }
     }
 
+    func clearDatabase(account: WalletConnectUtils.Account) {
+        subscriptions = subscriptions.filter { $0.account != account }
+    }
```

Comment:
does it clear anything?

Reply:
it's clear subscriptions for account, we're not care about messages in MockNotifyStoring.swift